src/components/file-upload/file-upload.ts

```diff
@@ -224,8 +229,8 @@ export class MFileUpload extends ModulVue {
         return this.$i18n.translate('m-file-upload:header-title', {}, this.maxFiles);
     }
 
-    private get fileExtensions(): string {
-        return this.extensions ? this.extensions.join(', ') : '';
+    private get fileAllowedExtensions(): string {
+        return this.allowedExtensions ? this.allowedExtensions.join(', ') : '';
```

Comment:
Avec la valeur par défaut, tu pourrais juste faire this.allowedExtensions.join(', ')